kong/plugins/rate-limiting/policies/cluster.lua

```diff
@@ -9,6 +9,27 @@ local floor = math.floor
 local fmt = string.format
 
 
+local EXPIRATION = require "kong.plugins.rate-limiting.expiration"
+
+
+local find
+do
+  local FIND_PK = {}
```

Comment:
This is to avoid doing one table allocation per call to `find`, right? I think that's ok, but the uppercase name suggests `FIND_PK` is a constant.

Reply:
changed to lowercase